sharding/notary/service.go

```diff
@@ -26,8 +26,7 @@ func (n *Notary) Start() error {
 	log.Info("Starting notary service")
 
 	// TODO: handle this better through goroutines. Right now, these methods
-	// have their own nested channels and goroutines within them. We need
-	// to make this as flat as possible at the Notary layer.
+	// are blocking.
```

Comment:
Could you please create a github issue for this so we don't forget? You can assign to me or leave unassigned or assign to yourself.

Reply:
Done with #152